guides/gatherers.md

```diff
@@ -66,3 +67,30 @@ facts:
 ```
 
 For extra information refer to [trento-project/agent/../gatherers/corosyncconf_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf_test.go)
+
+
+### package_version
+
+This gatherer allows checking the installation of a certain package in the system
+
+Sample arguments
```

Comment:
I would sincerely remove this `sample arguments` entry. It is quite obvious how to use it in the further examples.

Reply:
I agree that for this gatherer, it's quite obvious, but having consistency between all gatherers sections is important IMO. It barely makes a difference in space, and consistent sections generally allow for faster reading and finding what you want.

Reply:
So put some real package name, if its an example.
I don't know, I find it so redundant...